site/phpstan-baseline.neon

```diff
@@ -10532,12 +10537,12 @@ parameters:
 			path: app/views/AutoGradingView.php
 
 		-
-			message: "#^Method app\\\\views\\\\AutoGradingView\\:\\:convertToAnonPath\\(\\) has parameter \\$gradeable with no type specified\\.$#"
```

Comment:
You might want to check with William, but I think he has commented for one of my PRs, and one of Anson's PRs that you might be able to fix some of these errors rather than ignoring them, but you should ask him for more info to double check.